### DISCUSSION TITLE
PinToTaskbar feature

### DISCUSSION THREAD

**Discussion by 8HoLoN** [farag2/Sophia-Script-for-Windows#168] [Q&A] [answered]
Hi,

I saw you coded PinToStart functionnality (Windows-10-Sophia-Script/Sophia/PowerShell 5.1/Sophia.psm1). I wonder if you manage to do the same for the "pin to task bar" functionnality ? (I tried with syspin.exe but doesn't work).

I saw here (https://pinto10blog.wordpress.com/2016/09/10/pinto10/) that you tried to use syspin.exe but end up making your own script.

**Comment by farag2** [comment, marked as answer]
Hello! Yes, I've succeeded in using the syspin app but switched to the pure PowerShell to avoid using any 3rd party tools.
Yes I can code for you a script to pin some shortcuts to the taskbar.
Just describe what shortcuts you want to pin and I will think it over. Did you have a Telegram? Better to chat via our group. Actually you have a request not an issue. :)
https://t.me/sophia_chat

**Comment by farag2** [comment]
I think it will be fun to make such script. 🏆

**Comment by 8HoLoN** [comment]
No I do not have a telegram, I have a discord.

If you think it's do-able it's great i'm also interested in the way it's done (I thought only hack works because microsoft blocks that functionnality)

**Comment by farag2** [comment]
I'm too old for Discord). So our group is the only option.

**Comment by 8HoLoN** [comment]
ok I'm looking for telegram if it's your prefered platform

**Comment by farag2** [comment]
👍

**Comment by farag2** [comment]
A little bit busy now. You write and I'll reply there.

**Comment by 8HoLoN** [comment]
ok done, I'm in the group in telegram, Tell me when you want discuss the subject

**Comment by 8HoLoN** [comment]
I answer you in pm, i can't answer in main discussion, but ok for your message.
